src/modules/events/calibration/airspeed_calibration.cpp

```diff
@@ -245,9 +250,10 @@ int do_airspeed_calibration(orb_advert_t *mavlink_log_pub)
 			}
 
 			if (calibration_counter % 500 == 0) {
-				calibration_log_info(mavlink_log_pub, "[cal] Create air pressure! (got %d, wanted: 50 Pa)", (int)diff_pres.differential_pressure_filtered_pa);
-				tune_neutral(true);
```

Comment:
tune_neutral disappeared? This should be kept.

Reply:
@dagar 
airspeed_calibration.cpp and calibration_routines.cpp are using following routines
tune_neutral,
tune_negetive,
tune_positve,
set_tune,
rgbled_set_color_and_mode

As the routines are used by commander as well, we have three options.
1. Have same routines in calibration, which will lead to duplication of code (This might need some testing of all the cases).
2. include "commander/commander_helper.h" in airspeed_calibration.cpp and calibration_routines.cpp for now.
3. commander_helper.cpp mainly has led and tune related code, move this code to someother location and use it in both module (commander and callibration).